src/Tizen.NUI.Components/Controls/Slider.Internal.cs

```diff
@@ -197,6 +198,18 @@ private void OnPanGestureDetected(object source, PanGestureDetector.DetectedEven
                 UpdateValue();
             }
 
+            if (e.PanGesture.State == Gesture.StateType.Started)
+            {
+                if (null != sliderSlidingStartedHandler)
+                {
+                    SliderSlidingStartedEventArgs args = new SliderSlidingStartedEventArgs();
+                    args.CurrentValue = curValue;
+                    sliderSlidingStartedHandler(this, args);
+                }
+
+                UpdateState(isFocused, true);
+            }
```

Comment:
Can this conditional move to 185 line without duplication?
For example, 
```suggestion
            if (e.PanGesture.State == Gesture.StateType.Started)
            {
                if (direction == DirectionType.Horizontal)
                {
                    currentSlidedOffset = slidedTrackImage.SizeWidth;
                }
                else if (direction == DirectionType.Vertical)
                {
                    currentSlidedOffset = slidedTrackImage.SizeHeight;
                }
                
                if (null != sliderSlidingStartedHandler)
                {
                    SliderSlidingStartedEventArgs args = new SliderSlidingStartedEventArgs();
                    args.CurrentValue = curValue;
                    sliderSlidingStartedHandler(this, args);
                }
                
                UpdateState(isFocused, true);
            }
```